web_src/js/features/repo-diff.js

```diff
@@ -87,7 +87,7 @@ function initRepoDiffConversationForm() {
         el.classList.add('tw-invisible');
       }
       $newConversationHolder.find('.dropdown').dropdown();
-      initCompReactionSelector($newConversationHolder);
+      initCompReactionSelector();
```

Comment:
Duplicate init would case undefined behaviors.

Reply:
Not sure this is actually duplicate because the other init is gated by an `if`, will check.

Reply:
It was previously needed because new comments in diff view were appended dynamically.

I managed to remove this call by binding the `click` listener on higher-up parent nodes now in https://github.com/go-gitea/gitea/pull/30453/commits/56cf6b5bcc99f565b1e25e53ab392a9fc1236d62.